src/debug/debug.proto

```diff
@@ -42,8 +42,30 @@ message DumpRequest {
   int64 limit = 2;
 }
 
+message SetLogLevelRequest {
+    enum LogLevel {
+      UNKNOWN = 0;
+      DEBUG = 1;
```

Comment:
Do these have to be sequential? Should we match the constants that zap provides for these.  In `slog` there is a power of 2s thing going on.

Reply:
I'm not sure why slog uses powers of 2.  Powers of 2 always seem like that pattern where you `ONE | TWO | FOUR` to set some bits.  Maybe they mark each entry with ERROR|WARN|INFO|DEBUG and then do level & DEBUG == DEBUG to test whether or not to log at level DEBUG?  But why do that when < works just as well.